hlf/chaincode/operation.md

```diff
@@ -0,0 +1,187 @@
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the current model. This error should only occur if the model changes while the old ledger state remains without modification.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.  
+       For detailed informations see [Input Checks](#Checks).
+
+
+### GetOperationData
+- ID = getOperationData
+- Send
+    - operationId :: String
+- Receive
+    - operationData :: [OperationData](#OperationData)
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLUnprocessableLedgerState",
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the current model. This error should only occur if the model changes while the old ledger state remains without modification.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes are not well formatted, they are listed in invalidParams.  
+       For detailed informations see [Input Checks](#Checks).
+
+### GetOperations
+- ID = getOperations
+- Send
+    - enrollmentId :: String (optional)
+    - state :: String (optional)
+- Receive
+    - operationsList :: List\<[OperationData](#OperationData)\>
+      - Description: Returns the full list of existing Operations matching the filter parameters.
+        If these parameters are empty, the exhaustive list of all existing Operations is returned.
+
+    - [GenericError](errors.md#GenericError) 
+      ```json
+      {
+        "type": "HLUnprocessableLedgerState",
+        "title": "The state on the ledger does not conform to the specified format"
+      }
+      ```
+      - Description: This error is returned, if the state of data on the ledger is not consistent with the current model. This error should only occur if the model changes while the old ledger state remains without modification.
+

```

Comment:
```suggestion
### GetOperations
- ID = getOperations
- Send
    - enrollmentId :: String (optional)
    - state :: String (optional)
- Receive
    - operationsList :: List\<[OperationData](#OperationData)\>
      - Description: Returns the full list of existing Operations matching the filter parameters.
        If these parameters are empty, the exhaustive list of all existing Operations is returned.

```
Remove faulty error. If a single operation could not be rea we stil want to return as many "working" operationData as possible